src/actions/delete.ts

```diff
@@ -32,15 +32,9 @@ export default class Delete implements Action {
           text: "",
         }));
 
-        const calculateChanges = new CalculateChanges(
-          editor,
-          [targets.map((target) => target.selection.selection)],
-          edits
-        );
-
-        await performDocumentEdits(editor, edits);
-
-        const [updatedSelections] = calculateChanges.calculateSelections();
+        const [updatedSelections] = await performDocumentEdits(editor, edits, [
+          targets.map((target) => target.selection.selection),
+        ]);
```

Comment:
was thinking of just baking this stuff into `performDocumentEdits` so that we don't have this multi-step process.  I already implemented it lmk what you think

Reply:
Sounds like a good solution.